style.css

```diff
@@ -1,468 +1,1187 @@
-*, *:before, *:after {
+*,
+*::before,
+*::after {
     padding: 0;
     margin: 0;
     border: 0;
     box-sizing: border-box;
 }
+
 body {
     margin: 0 auto;
     max-width: 1440px;
     font-family: 'Montserrat', sans-serif;
     font-style: normal;
-    background: #FFFFFF;
+    background: #fff;
 }
+
 .header {
-    margin: 0 auto;
     display: flex;
-    width: 1337.28px;
-    height: 51px;
-    padding: 0;
-    margin-top: 24px;
-    margin-left: 42.7px;
+    justify-content: space-between;
+    width: 100%;
+    padding: 25px 60px 0 41px;
 }
+
 .header__logo {
     display: flex;
     width: 166px;
     height: 51px;
 }
+
 .header__menu {
     display: flex;
     align-items: center;
-    padding-left: 258px;
-    top: 2px;
+    justify-content: space-between;
+    width: 490px;
+    margin-left: 38px;
 }
+
 .menu__link {
     text-decoration: none;
     font-weight: 600;
     font-size: 15px;
-    line-height: 18px;
+    line-height: 33px;
     color: #000000;
 }
-.menu__link:nth-child(2) {
-    padding-left: 63px;
-    padding-right: 25px;
-}
-.menu__link:nth-child(4) {
-    padding-left: 45px;
-    padding-right: 46px;
-}
-.menu__link:nth-child(5) {
-   padding-right: 221px;
-}
-/*.menu__link:hover {*/
-/*    color: #e51937;*/
-/*    border-bottom: 2px solid #e51937;*/
-/*}*/
+
 .menu__link__active {
     color: #e51937;
     border-bottom: 2px solid #e51937;
 }
+
+.header__menu-second {
+    width: 205px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
 .header__button {
     background-color: transparent;
-    font-family: 'Montserrat', sans-serif;
+    font-family: "Montserrat", sans-serif;
     font-style: normal;
     font-weight: 600;
     font-size: 15px;
     color: #000000;
-    padding-top: 17px;
-    padding-left: 30px;
+    align-items: center;
+    line-height: 30px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)